projects/cmake/CMakePresets.json

```
@@ -1,60 +1,76 @@
 {
     "version": 8,
+    "cmakeMinimumRequired": {
+      "major": 3,
+      "minor": 10,
```

CMake presets were added in version 3.19 so this is practically the minimum
```suggestion
      "minor": 19,
```